<?xml version="1.0" encoding="UTF-8"?>
<!-- Generated by org.testng.reporters.JUnitXMLReporter -->
<testsuite ignored="0" hostname="MouriceSeraphin" failures="1" tests="6" name="Test" time="23.522" errors="0" timestamp="2025-05-23T12:31:24 IST">
  <testcase classname="todotests.TC_005ValidateFinalTaskCountAndStatus" name="validateFinalTaskCountAndStatus" time="0.953"/>
  <testcase classname="todotests.TC_003MarkSecondTaskAsCompleted" name="markSecondTaskAsCompleted" time="0.796"/>
  <testcase classname="todotests.TC_001OpenApp" name="openApplication" time="0.022"/>
  <testcase classname="todotests.TC_004DeleteFirstTask" name="deleteFirstTask" time="0.765"/>
  <testcase classname="todotests.TC_002AddThreeTask" name="addThreeTasks" time="0.533"/>
  <testcase classname="todotests.TC_006NegativeTCDeleteInvalidIndex" name="deleteNonExistentTask" time="0.189">
    <failure type="java.lang.AssertionError" message="Invalid index provided: 1000. No such task exists.">
      <![CDATA[java.lang.AssertionError: Invalid index provided: 1000. No such task exists.
at utilities.TestUtil.logFailAndFailTest(TestUtil.java:58)
at todotests.TC_006NegativeTCDeleteInvalidIndex.deleteNonExistentTask(TC_006NegativeTCDeleteInvalidIndex.java:17)
at java.base/java.util.ArrayList.forEach(ArrayList.java:1511)
... Removed 27 stack frames]]>
    </failure>
  </testcase> <!-- deleteNonExistentTask -->
</testsuite> <!-- Test -->
